tests/lax_numpy_test.py

```diff
@@ -3547,8 +3542,11 @@ def testLogspace(self, start_shape, stop_shape, num,
     for axis in range(-ndim, ndim):
       jnp_op = lambda start, stop: jnp.logspace(
         start, stop, num, endpoint=endpoint, base=base, dtype=dtype, axis=axis)
-      np_op = lambda start, stop: np.logspace(
-        start, stop, num, endpoint=endpoint, base=base, dtype=dtype, axis=axis)
+      @jtu.ignore_warning(category=RuntimeWarning,
+                          message="overflow encountered in power")
```

Comment:
For my understanding, what is this for? Does the test fail without this somehow?

Reply:
The rearranging of types in the parametrization caused our tests to exercise a different set of randomly generated inputs, which happened to trigger an overflow here. Because our test scaffold fails on warnings, this was necessary to prevent flakes.